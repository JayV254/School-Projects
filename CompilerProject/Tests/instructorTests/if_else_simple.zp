
if main() {
	int a;
	int b;
	int c;
	int d;

	a = 1;
	b = 2;
	c = 3;
	d = 4;

	if (a < b) {
		puts("a is less than b");
	}
	else {
		puts("b is less than a");
	}

	if (d < c) {
		puts("d is less than c");
	}
	else {
		puts("c is less than d");
	}

	return 0;
}
