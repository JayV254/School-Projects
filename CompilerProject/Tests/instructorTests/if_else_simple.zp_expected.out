a is less than b
c is less than d
